File: IQ/Classes/clsVisibility.vb
Public Class clsVisibility

    Public path As String
    Public hideReasonList As List(Of String)
    Public branch As clsBranch

    Public Sub New(branch As clsBranch, path As String, HideReasonList As List(Of String))
        Me.path = path
        Me.hideReasonList = HideReasonList
        Me.branch = branch

    End Sub

End Class

File: IQ/Classes/clsVisibility.cs
public class clsVisibility
{

    public string path;
    public List<string> hideReasonList;
    public clsBranch branch;

    public clsVisibility(clsBranch branch, string path, List<string> HideReasonList)
    {
        this.path = path;
        this.hideReasonList = HideReasonList;
        this.branch = branch;

    }

}